scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/constraints/activities/ActivityExpression.java

```diff
@@ -334,6 +334,7 @@ Builder basedOn(@NotNull ActivityExpression template) {
       endsIn = template.endRange;
       durationIn = template.durationRange;
       startsOrEndsIn = template.startOrEndRange;
+      startsOrEndsInW = template.startOrEndRangeW;
```

Comment:
Do you recall what the difference is between `startsOrEndsIn` and `startsOrEndsInW`? Not essential for this PR - it just strikes me as a possible point of confusion.

Reply:
Yes, one is just for one interval, the other for a set of intervals. One is reducible to the other. I am not why we have both but I believe this should be cleaned when we are going to work on the activity search vs activity creation.